.travis.yml

```diff
@@ -13,6 +13,7 @@ bundler_args: --without development:production --deployment --retry=3 --jobs=3
 before_script:
   - bundle exec rake db:create
   - bundle exec rake db:migrate
+  - bundle exec rake assets:precompile &> /dev/null
```

Comment:
> According to [teampoltergeist/poltergeist#677 (comment)](https://github.com/teampoltergeist/poltergeist/issues/677#issuecomment-222919584), a possible cause of the "Request failed to reach server" problem is that assets are being compiled on-demand, causing enough of a slowdown for PhantomJS to think the connection has failed. This turns off lazy asset compilation in the test environment and precompiles assets before testing on Travis.

I encountered this [problem on Travis](https://travis-ci.org/ruby-bench/ruby-bench-web/builds/250922521#L622) and solved it with assets precompilation.

Error occurred most probably because of introducing admin layout with its own asset group.